src/config/gIndex.config.ts

```diff
@@ -39,8 +39,8 @@ const config: gIndexConfig = {
      * You need to create a new folder and share it with the service account
      * Then, copy the folder id and paste it here
      */
-    rootFolder: "1KgPV6QB1GYT8fmn2uTfbtr9rDXqcRR0j",
-
+    rootFolder: "",
```

Comment:
Can you revert the root folder id to the previous value so it doesn't break the current demo site?
After that i will approve the PR

Reply:
ok sure